### PR TITLE
chore: prepare bottlerocket-settings-model-v0.3.0

### PR DESCRIPTION
**Pull request by cbgbt**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: bottlerocket-settings-models/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+
+- Skipped serializing credential provider fields if they are None ([#51])
+- Moved kubernetes models to a kubernetes settings extension ([#53])
+- Updated dependencies ([#50], [#47])
+
+[#43]: https://github.com/bottlerocket-os/bottlerocket-settings-sdk/pull/43
+[#47]: https://github.com/bottlerocket-os/bottlerocket-settings-sdk/pull/47
+[#50]: https://github.com/bottlerocket-os/bottlerocket-settings-sdk/pull/50
+[#51]: https://github.com/bottlerocket-os/bottlerocket-settings-sdk/pull/51
```
Comment (sam-berning) [inline bottlerocket-settings-models/CHANGELOG.md:26]: missing the link for #53 here

Comment (cbgbt) [reply bottlerocket-settings-models/CHANGELOG.md:26]: Whoops! Nice catch.

**Comment by cbgbt** [comment]
^ force push to add missing link pointed out by @sam-berning

**Review by sam-berning — APPROVED** [review]

**Comment by cbgbt** [comment]
^ force push to bump crate versions

**Review by rpkelly — APPROVED** [review]

**Review by jpculp — APPROVED** [review]

**Review by bcressey — APPROVED** [review]

**Pull request merged**